migrations/20240404084539_nationalityupdate_again.up.sql

```diff
@@ -0,0 +1,461 @@
+    ('NOR', 'Normandy', 'FR'),
+    ('GES', 'Grand Est', 'FR'),
+    ('PDL', 'Lands of the Loire', 'FR'),
+    ('BRE', 'Brittany', 'FR'),
+    ('BFC', 'Burgundy-Free County', 'FR'),
+    ('CVL', 'Centre-Loire Valley', 'FR'),
+    ('ARA', 'Auvergne-Rhone-Alpes', 'FR'),
+    ('NAQ', 'New Aquitaine', 'FR'),
+    ('20R', 'Corsica', 'FR'),
+    ('PM', 'Saint Pierre and Miquelon', 'FR'),
+    ('BL', 'Saint Barthelemy', 'FR'),
+    ('MF', 'Saint Martin', 'FR'),
+    ('NC', 'New Caledonia', 'FR'),
+    ('WF', 'Wallis and Futuna', 'FR'),
+    ('PF', 'French Polynesia', 'FR'),
+    ('21', 'Piedmont', 'IT'),
+    ('23', 'Aosta Valley', 'IT'),
+    ('25', 'Lombardy', 'IT'),
+    ('32', 'Trentino-Alto Adige', 'IT'),
+    ('34', 'Veneto', 'IT'),
+    ('36', 'Friuli-Venezia Giulia', 'IT'),
+    ('42', 'Liguria', 'IT'),
+    ('45', 'Emilia-Romagna', 'IT'),
+    ('52', 'Tuscany', 'IT'),
+    ('55', 'Umbria', 'IT'),
+    ('57', 'Marche', 'IT'),
+    ('62', 'Lazio', 'IT'),
+    ('65', 'Abruzzo', 'IT'),
+    ('67', 'Molise', 'IT'),
+    ('72', 'Campania', 'IT'),
+    ('75', 'Apulia', 'IT'),
+    ('77', 'Basilicata', 'IT'),
+    ('78', 'Calabria', 'IT'),
+    ('82', 'Sicily', 'IT'),
+    ('88', 'Sardinia', 'IT'),
+    ('ML', 'Melilla', 'ES'),
+    ('CE', 'Ceuta', 'ES'),
+    ('CL', 'Castile and Leon', 'ES'),
+    ('PV', 'Basque Country', 'ES'),
+    ('IB', 'Balearic Islands', 'ES'),
+    ('CN', 'Canary Islands', 'ES'),
+    ('CT', 'Catalonia', 'ES'),
+    ('VC', 'Valencian Community', 'ES'),
+    ('MC', 'Region of Murcia', 'ES'),
+    ('AN', 'Andalusia', 'ES'),
+    ('CM', 'Castilla–La Mancha', 'ES'),
+    ('EX', 'Extremadura', 'ES'),
+    ('AR', 'Aragon', 'ES'),
+    ('MD', 'Community of Madrid', 'ES'),
+    ('NA', 'Navarre', 'ES'),
+    ('RI', 'La Rioja', 'ES'),
+    ('GA', 'Galicia', 'ES'),
+    ('AS', 'Asturias', 'ES'),
+    ('CB', 'Cantabria', 'ES'),
+    ('HI', 'Hainan', 'CN'),
+    ('NM', 'Inner Mongolia Autonomous Region', 'CN'),
+    ('HL', 'Heilongjiang', 'CN'),
+    ('MO', 'Macao Special Administrative Region', 'CN'),
+    ('JL', 'Jilin', 'CN'),
+    ('LN', 'Liaoning', 'CN'),
+    ('GS', 'Gansu', 'CN'),
+    ('BJ', 'Beijing', 'CN'),
+    ('TJ', 'Tianjin', 'CN'),
+    ('HE', 'Hebei', 'CN'),
+    ('SD', 'Shandong', 'CN'),
+    ('JS', 'Jiangsu', 'CN'),
+    ('SX', 'Shanxi', 'CN'),
+    ('HA', 'Henan', 'CN'),
+    ('SH', 'Shanghai', 'CN'),
+    ('ZJ', 'Zhejiang', 'CN'),
+    ('SN', 'Shaanxi', 'CN'),
+    ('HB', 'Hubei', 'CN'),
+    ('JX', 'Jiangxi', 'CN'),
+    ('FJ', 'Fujian', 'CN'),
+    ('HN', 'Hunan', 'CN'),
+    ('SC', 'Sichuan', 'CN'),
+    ('CQ', 'Chongqing', 'CN'),
+    ('YN', 'Yunnan', 'CN'),
+    ('GZ', 'Guizhou', 'CN'),
+    ('GX', 'Guangxi Zhuang Autonomous Region', 'CN'),
+    ('GD', 'Guangdong', 'CN'),
+    ('NX', 'Ningxia Hui Autonomous Region', 'CN'),
+    ('QH', 'Qinghai', 'CN'),
+    ('XZ', 'Tibet Autonomous Region', 'CN'),
+    ('XJ', 'Xinjiang Uygur Autonomous Region', 'CN'),
+    ('AH', 'Anhui', 'CN'),
```

Comment:
China only has 10 players, which is less than this number of subdivisions. Let's not bother with subdivisions here.